geode-core/src/main/java/org/apache/geode/internal/cache/backup/FinishBackup.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.internal.cache.backup;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashSet;
+
+import org.apache.geode.cache.persistence.PersistentID;
+import org.apache.geode.internal.cache.InternalCache;
+
+class FinishBackup {
+
+  private final InternalCache cache;
+  private final File targetDir;
+  private final File baselineDir;
+  private final boolean abort;
+
+  FinishBackup(InternalCache cache, File targetDir, File baselineDir, boolean abort) {
+    this.cache = cache;
+    this.targetDir = targetDir;
+    this.baselineDir = baselineDir;
+    this.abort = abort;
+  }
+
+  HashSet<PersistentID> run() throws IOException {
```

Comment:
I remember from previously looking through this code that we have to return a HashSet, because the serializer we have does not support serializing a Set (or rather it does, but that method is private to the serializer). This is outside the scope of this PR, but we should consider if it is viable to change this behavior in the future. As long as a special order in not needed when it it deserialized, there does not appear to be a need to serialize a specific type of map.